adserver/tasks.py

```diff
@@ -581,6 +590,57 @@ def notify_of_completed_flights():
             flight.live = False
             flight.save()
 
+            completed_flights_by_advertiser[flight.campaign.advertiser.slug].append(
+                flight
+            )
+
+    # Send notification to advertiser - one email even if multiple flights finished
+    if settings.FRONT_TOKEN and settings.FRONT_AUTHOR and settings.FRONT_CHANNEL:
+
+        site = get_current_site(request=None)
+
+        for (
+            advertiser_slug,
+            completed_flights,
+        ) in completed_flights_by_advertiser.items():
+            advertiser = Advertiser.objects.get(slug=advertiser_slug)
+
+            to_addresses = [
+                u.email
+                for u in advertiser.user_set.all()
+                if u.notify_on_completed_flights
+            ]
+            context = {
+                "advertiser": advertiser,
+                "site": site,
+                # Just the flights completed today
+                "completed_flights": completed_flights,
+                "current_flights": [
+                    f
+                    for f in Flight.objects.filter(campaign__advertiser=advertiser)
+                    if f.state == FLIGHT_STATE_CURRENT
+                ],
+                "upcoming_flights": [
+                    f
+                    for f in Flight.objects.filter(campaign__advertiser=advertiser)
+                    if f.state == FLIGHT_STATE_UPCOMING
+                ],
+            }
+
+            with mail.get_connection(settings.FRONT_BACKEND) as connection:
+                message = mail.EmailMessage(
+                    _("Advertising flight wrapup - %(name)s") % {"name": site.name},
+                    render_to_string("adserver/email/flight_wrapup.html", context),
+                    settings.SERVER_EMAIL,  # Front doesn't use this
+                    to_addresses,
```

Comment:
I always prefer kwargs if possible on these, so we know what these arguments are.